src/views/ProtestPage.js

```diff
@@ -29,6 +29,7 @@ import {
   isVisitor,
   sortDateTimeList,
 } from '../utils';
+import LoadingSpinner from '../components/LoadingSpinner/LoadingSpinner';
```

Comment:
You can omit the last part
```suggestion
import LoadingSpinner from '../components/LoadingSpinner';
```